src/cpp/shared/gui/msg.cpp

```diff
@@ -31,3 +35,13 @@ void GUIBroadcastMessage(heroWindow* hwnd, int f, int c, void* p) {
 	evt.payload = p;
 	hwnd->BroadcastMessage(evt);
 }
+
+int GUIGetDropdownSelection(heroWindow* hwnd, void* p) {
+  tag_message evt;
+  evt.yCoordOrFieldID = 100;
+  evt.eventCode = INPUT_GUI_MESSAGE_CODE;
+  evt.xCoordOrKeycode = 55;
+  evt.payload = p;
```

Comment:
It is the value of the spell that is being selected in a dropdown. I guess that I could define it as a constant; I did not do this previously because I was following the conventions used in the previous GUI helper functions.